docker/docker-bake.hcl

```diff
@@ -79,11 +79,7 @@ target "editoast-test" {
   }
 }
 
-target "base-editoast" {
-  args = {
-    CARGO_FEATURES = "datadog"
-  }
-}
```

Comment:
Should I also remove `CARGO_FEATURE` in the editoast `Dockerfile` ?

Reply:
I'd say that if it's easy to remove, let's remove it, the code be less cluttered and I don't see a short-term opportunity to use that functionality anyway.